src/components/dialogs/load-creation-dialog.js

```diff
@@ -197,186 +212,292 @@ const LoadCreationDialog = ({
         }
     };
 
+    const [isDialogSearchOpen, setDialogSearchOpen] = useState(false);
+
+    const handleOpenSearchDialog = () => {
+        setDialogSearchOpen(true);
+    };
+
+    const handleSelectionChange = (element) => {
+        let msg;
+        fetchLoadInfos(studyUuid, workingNodeUuid, element.id).then(
+            (response) => {
+                if (response.status === 200) {
+                    response.json().then((load) => {
+                        //For now we don't want to retrieve nor try to set the BusBarSection, users have to select it.
+                        setLoadId(load.id);
```

Comment:
no ' (1)' added to the load id ?

Reply:
need to talk about that friday as the field id might disappear from these kind of form